windows_10.json

```diff
@@ -65,7 +65,9 @@
         "./scripts/chocolatey.bat",
         "./scripts/chocolatey-packages.bat",
         "./scripts/firewall.bat",
-        "./scripts/doit.bat"
+        "./scripts/doit.bat",
+        "./scripts/clean.bat",
+        "./upstream/scripts/compact.bat"
```

Comment:
Yeah, it is nice to have a smaller image. While these builds were running, we noticed there was almost no actual disk activity (the VM would be showing a few KB/s, nothing was being done). It was very odd.

If everything is working fine with compact, I think we are fine. It's just a major pain to build images that takes close to 2 hours to build. Maybe we can remember to take compact.bat out when building the first images and then drop it back in when we confirm everything is okay. But that'd be up to the person that's building the image at that time.

I just wanted to check everything was fine with introducing it. Thanks for the numbers!